en/schemas.html

```diff
@@ -257,5 +255,32 @@ <h2 id="content-cluster-mapping">Content cluster mapping</h2>
 </p>
 
 
-<!-- ToDo: A section here on how to modify the schema,
-           link to https://docs.vespa.ai/en/reference/schema-reference.html#modifying-schemas -->
+
+<h2 id="schema-modifications">Schema modifications</h2>
+<p>
+  Vespa is built for safe schema modifications,
+  like adding a field or changing indexing or match modes.
+  A new version of the schema is deployed in an <a href="cloudconfig/application-packages.html">application package</a>.
+  As some changes are potentially destructive (e.g. remove a field) or requires more actions (e.g. restart),
+  the deploy will by default not accept such changes - example:
+</p>
+<pre>
+WARNING: Change(s) between active and new application that require restart:
+In cluster 'items' of type 'search':
+    Restart services of type 'searchnode' because:
+        1) Document type 'music': Field 'title' changed: add attribute aspect
+</pre>
+<p>
+  To accept such changes, add a <a href="reference/validation-overrides.html">validation-override</a>, e.g.
+</p>
+<pre>
+&lt;validation-overrides&gt;
+    &lt;allow until="2021-08-30"&gt;indexing-change&lt;/allow&gt;
+&lt;/validation-overrides&gt;
+</pre>
+<p>
```

Comment:
I don't think a change leading to a restart is a good example, the change will happen anyway and there is no validation override for restart

Reply:
Good point! modifying to a field indexing change instead